web-app/README.md

```diff
@@ -1,3 +1,195 @@
+
+By the end of the script, there should be a new version of the `kiosk.zip` file inside of the `web-app/` folder.
+
+## Upload the ZIP to a Cloud 
+
+This new ZIP file for the kiosk created in the previous section needs to be uploaded to a repository where the Notehub cloud can access it via HTTP. For this example, an [Amazon Web Services S3 repo](https://aws.amazon.com/s3/) was used.
+
+### Create New S3 Bucket
+
+Inside of the AWS Console, navigate to the Amazon S3 bucket and click the **Create Bucket** button in the upper right hand corner of the screen.
+
+**Name the bucket** 
+
+Give the bucket an easily recognizable name.
+
+![Name new AWS bucket](images/aws-create-bucket.png)
+
+**Enable the bucket's ACLs**
+
+Make the bucket's contents ownable by other AWS accounts.
+
+![Select bucket's ownership option](images/aws-bucket-ownership.png)
+
+**Make the bucket publicly accessible**
+
+In order for Notehub to download the ZIP file from the bucket, the bucket will need to be publicly accessible.
+
+Amazon warns this is a dangerous move, but uncheck the **Block all public access** check box and acknowledge the bucket and its contents could become public.
+
+![Allow bucket to be accessed by the public](images/aws-public-bucket.png)
+
+### Upload the ZIP to the S3 Bucket
+
+Now that the bucket is created, click on its name in the list of buckets and upload the ZIP file to it by clicking the **Upload** button.
+
+**Add files to the bucket**
+
+On the Upload screen, click the **Add files** button and select the locally created `kiosk.zip` file.
+
+![Upload the ZIP file to the bucket](images/aws-upload-zip.png)
+
+**Grant public-read access in permissions**
+
+Open the **Permissions** dropdown beneath where you selected the file, and select the **Grant public-read access** radio button, once again, Amazon will let you know this is risky, but acknowledge the risk because Notehub needs to be able to reach and download the contents of this bucket.
+
+![Make the ZIP file accessible to the public in the bucket](images/aws-public-zip.png)
+
+Then hit the **Upload** button at the end of the page.
+
+If all goes well, you'll see an Upload Status page with a success message confirming the ZIP file was successfully uploaded. Don't close this tab yet though, you'll need some additional information from it shortly for Notehub.
+
+![Success message when file uploaded successfully to bucket](images/aws-successful-upload.png)
+
+## Create a Notehub Proxy Route Pointing Towards the AWS Bucket
+
+At this point, you should have already created a Notehub project for this app from following along with the [`README.md` documentation](https://github.com/blues/app-accelerators/blob/main/27-cellular-connected-electronic-kiosk/README.md) in the main app-accelerators repo, if not, go ahead and do so now.
```

Comment:
And here:

> repo. If not, go ahead

Reply:
You're good at catching my run on sentences 😄